pkg/koordlet/util/system/resctrl.go

```diff
@@ -256,11 +256,18 @@ func (r *ResctrlSchemataRaw) L3Number() int {
 }
 
 func (r *ResctrlSchemataRaw) CacheIds() []int {
```

Comment:
Would you be able to separate the L3 number and the MB number? Only using the larger one may cause the wrong output in the DeepCopy and String methods.

Reply:
Yes, then we can split this one.

Reply:
OK. Please leave a TODO here and resolve it later.
Currently, there are some methods invoking the `CacheIds()` like `ResctrlSchemataRaw.DeepCopy()` and `MBString()`, it only supports the case where the L3 number and the MB number are the same.

Reply:
yes, we can only support the case which L3 number == MB number. We leave a TODO here and resolve it later.

Reply:
> OK. Please leave a TODO here and resolve it later. Currently, there are some methods invoking the `CacheIds()` like `ResctrlSchemataRaw.DeepCopy()` and `MBString()`, it only supports the case where the L3 number and the MB number are the same.

Add a TODO here. We could fix it in other PR.